frontend/src/modules/entitieslist/components/EntitiesList.js

```diff
@@ -218,8 +218,13 @@ export class EntitiesListBase extends React.Component<InternalProps> {
     };
 
     getSiblingEntities = (entity: number) => {
+        const entityIds = this.props.entities.entities.map(
+            (entity) => entity.pk,
+        );
```

Comment:
That could be a lot of entities, which means a lot of data, and a big loop in the action to check if the list of entities contains any slings.

It should be enough (and scale much better) if you just pass the immediate preceding and succeeding entity.

Reply:
Can I have more context on how to implement this, I thought we needed to use the previous list to filter out the `siblingsEntities`?

Reply:
I think you could do something even simpler, at least as the first step: you could just remove duplicates at the end of `injectSiblingEntities`.